x-pack/plugins/apm/public/components/app/help_popover/help_popover.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { ReactNode } from 'react';
+import {
+  EuiButtonIcon,
+  EuiLinkButtonProps,
+  EuiPopover,
+  EuiPopoverProps,
+  EuiPopoverTitle,
+  EuiText,
+} from '@elastic/eui';
+import { euiStyled } from '../../../../../../../src/plugins/kibana_react/common';
+
+const PopoverContent = euiStyled(EuiText)`
+  max-width: 480px;
+  max-height: 40vh;
+`;
+
+export function HelpPopoverButton({
+  onClick,
+}: {
+  onClick: EuiLinkButtonProps['onClick'];
+}) {
+  return (
+    <EuiButtonIcon
+      className="apmHelpPopover__buttonIcon"
+      size="s"
+      iconType="help"
+      aria-label="Help"
```

Comment:
This text should be internationalized.
Also I noticed that the `EuiButtonIcon` in the ML help popover is missing the `aria-label` prop, which is causing a lot of warnings to be logged to the browser console. Could you please add an `aria-label` prop to the `EuiButtonIcon` in `ml/public/application/components/help_popover/help_popover.tsx`.

Reply:
Good catch!  I've updated that aria-label in 3f718962047723188f70cdf025e06da345c20769.  I'll see if I can get the ML popover fixed in this PR too, otherwise I'll do it in a separate PR.

Reply:
I've addressed the ML popover (and fixed the missing imports) in d875c3e4188b859bcaea3529f5d610533cca1040